dbptk-core/src/main/java/com/databasepreservation/modules/siard/out/metadata/FileIndexFileStrategy.java

```diff
@@ -18,8 +18,16 @@
 import com.databasepreservation.model.exception.ModuleException;
 import com.databasepreservation.model.structure.DatabaseStructure;
 import com.databasepreservation.modules.siard.common.SIARDArchiveContainer;
+import com.databasepreservation.modules.siard.constants.SIARDDKConstants;
+import com.databasepreservation.modules.siard.out.output.SIARDDKExportModule;
 import com.databasepreservation.modules.siard.out.write.WriteStrategy;
-import dk.magenta.siarddk.fileindex.FileIndexType;
+
+// import dk.magenta.siarddk.fileindex.FileIndexType;
+
+
+
+
```

Comment:
No - forgot to remove those

2015-10-12 10:29 GMT+02:00 Bruno Ferreira <notifications@github.com>:

> In
> dbptk-core/src/main/java/com/databasepreservation/modules/siard/out/metadata/FileIndexFileStrategy.java
> <https://github.com/keeps/db-preservation-toolkit/pull/74#discussion_r41733848>
> :
>
> >  import com.databasepreservation.modules.siard.out.write.WriteStrategy;
> > -import dk.magenta.siarddk.fileindex.FileIndexType;
> > +
> > +// import dk.magenta.siarddk.fileindex.FileIndexType;
> > +
> > +
> > +
> > +
>
> is there any special reason to keep the comment and blank lines?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keeps/db-preservation-toolkit/pull/74/files#r41733848>
> .
>



-- 
Magenta Aps

E-mail: andreas@magenta.dk
Tlf.: 81 77 16 98